tests/fuzzFail/fuzz_app_file_fail40

```diff
@@ -0,0 +1 @@
+     config                      ' ' 
```

Comment:
Haven't followed the fuzzer that closely, what does this do?

Reply:
Right now the fuzzer has a large number of options,  and some limited mechanics to generate new ones.  It processes the options making sure there is nothing that generates a seg fault or other bad condition.   Then generates a config file from the results and reads that config file back in.      This led to all kinds of inconsistencies that got fixed a few months ago.   At some point I will make it so it checks to make sure the values read from the config file are actually the same as the original values, then I might consider submitting it to [OSS-Fuzz](https://github.com/google/oss-fuzz).   The current test just runs a quick_fuzz ~6 minutes and can catch some issues especially when changing some internal mechanics or other things.    I will keep adding to it as I have time.  

This particular test caught an issue with creating a positional option named config, which was newly allowed.  This then triggered an error as it was trying to match with the config option which was not configurable, so had to make it so it checked for other options if some were not configurable.